testcases/misc_tests/p11sak_test.sh

```diff
@@ -137,6 +147,26 @@ echo "* TESTCASE remove-key des FAIL Failed to delete generated DES key"
 fi
 
 
+# CK_BBOOL
+if [[ $(grep -A 19 'p11sak-des' $P11SAK_DES_LONG | grep -c 'CKA_IBM_PROTKEY_EXTRACTABLE: CK_FALSE') == "1" && $rc = 0 ]]; then
```

Comment:
I don't understand the `$rc = 0` part here.  If that is true we know that the `p11sak-des` key no longer exists after removal.  What does this have to do with the listing here?

Reply:
Removed